### DISCUSSION TITLE
Event Presets

### DISCUSSION THREAD

**Discussion by Baxuvis** [4ian/GDevelop#6414] [Enhancement ideas (and their technical discussions)]
### Feature Suggestion ( Event Presets ) ###
This one is short and sweet. 
A feature suggestion, to Save (Export) and (Import) Events or Parts of events. That way you can load them into the Events quickly rather than creating something, going to a different project to copy and paste it. I have a few Events I've coded that took me a while to figure out and I would love to be able to save it as a preset for other projects and then once imported, just rename the objects they're looking for and import whatever extensions they needed. This I believe would be a very wonderful Ease of Access Feature that Could save many people Lots of time and also Enable Easier Sharing of Code to newer Developers in GDev.

**Comment by 4ian** [comment]
Hi! 
The way to do this in GDevelop is by creating an extension and adding an action that contains these events. This way you can copy/paste the extension to another project and call this action containing your events (and even pass parameters to adapt what they do).
Note that you can select events in the Events Sheet, do a right click, choose Replace > Extract Events to a Function, that will do this for you after a bit of setup :)
